### ISSUE TITLE
Oryp6 - Nvidia freezes with "Can't change power state from D3cold to D0"

### ISSUE THREAD

**Issue by JoonasC** [system76/firmware-open#259]
Sometimes my laptop will completely freeze with the following error logs in journalctl:
```
kernel: nvidia 0000:01:00.0: can't change power state from D3cold to D0 (config space inaccessible)
kernel: NVRM: GPU at PCI:0000:01:00: GPU-a137dd66-f6a3-7951-1348-c37443697050
kernel: NVRM: Xid (PCI:0000:01:00): 79, pid=7992, GPU has fallen off the bus.
kernel: NVRM: GPU 0000:01:00.0: GPU has fallen off the bus.
kernel: xhci_hcd 0000:01:00.2: can't change power state from D3cold to D0 (config space inaccessible)
kernel: xhci_hcd 0000:01:00.2: can't change power state from D3hot to D0 (config space inaccessible)
kernel: xhci_hcd 0000:01:00.2: Controller not ready at resume -19
kernel: xhci_hcd 0000:01:00.2: PCI post-resume error -19!
kernel: xhci_hcd 0000:01:00.2: HC died; cleaning up
kernel: pci 0000:01:00.3: can't change power state from D3cold to D0 (config space inaccessible)
kernel: snd_hda_intel 0000:01:00.1: can't change power state from D3cold to D0 (config space inaccessible)
kernel: snd_hda_codec_hdmi hdaudioC1D0: Unable to sync register 0x4f0800. -5
```

This seems to only occur on hybrid graphics mode.
Probably related to #160.

**Comment by crawfxrd** [comment]
Can you reliably reproduce this? At what time after boot do those logs appear? (If you still have the journal, you can use `-o short-monotonic` to get timestamps like dmesg.)

**Comment by JoonasC** [comment]
I can't really reproduce the issue reliably, some days and even weeks it would work just fine without problems. There doesn't really seem to be a fixed time period after boot when it happens either. Once it froze straight on boot, another time after exiting sleep mode and another time after the computer had already been running for some time.

**Comment by crawfxrd** [comment]
Should be fixed with updated coreboot in fd9d73bdb9c8.

**Comment by crawfxrd** [comment]
Duplicate of #160
